manifests/m/MacroDeck/MacroDeck/2.12.0/MacroDeck.MacroDeck.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-22572-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: MacroDeck.MacroDeck
+PackageVersion: 2.12.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+UpgradeBehavior: install
+InstallerSwitches:
+  Silent: /VERYSILENT /SUPPRESSMSGBOXES /CLOSEAPPLICATIONS
+  SilentWithProgress: /SILENT /SUPPRESSMSGBOXES /CLOSEAPPLICATIONS
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Macro-Deck-App/Macro-Deck/releases/download/v2.12.0/macro-deck-2.12.0-win-x64.exe
```

```suggestion
Installers:
- Architecture: x64
  InstallerUrl: https://github.com/Macro-Deck-App/Macro-Deck/releases/download/v2.12.0/macro-deck-2.12.0-win-x64.exe
  AppsAndFeaturesEntries:
  - DisplayVersion: 2.12.0.221941808
```
This is needed because PackageVersion does not match control panel